community_supplied/Platform/platform.hbm_hbm.rule

```diff
@@ -0,0 +1,81 @@
+healthbot {
```

Comment:
Add the rule functionality in the form of comments i.e.
/*
 * <comment1>
 * <commentN>
 * /

Refer rule for more information on comments: https://github.com/Juniper/healthbot-rules/blob/master/juniper_official/Interfaces/interface-flaps.rule